### PR TITLE
Don't fail the job when input term is empty

### PR DESCRIPTION
**Pull request by annyan09023**
Unit test is changed because I added an empty string to the original text.avro. It caused another tensorMetadata generation test change because empty string is a new term

### CODE REVIEW

File: avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/FeatureListGeneration.scala [hunk 1]
```diff
@@ -300,7 +300,8 @@ object FeatureListGeneration {
                     // the format of each line is feature_entry,count
                     // first get feature_entry, if need process prefix (ntv), remove prefix from feature_entry, make
                     // sure prefix is unique
-                    val lineWithoutCount = line.split(SPLIT_REGEX).head
+                    val words = line.split(SPLIT_REGEX)
+                    val lineWithoutCount = if (words.isEmpty) "" else words.head
```
Comment (zhangxuhong) [inline avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/FeatureListGeneration.scala:304]: can you also take a look at line 376, might have similar bug

Comment (annyan09023) [reply avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/FeatureListGeneration.scala:304]: okay, I added it and added an unit test for it. Thanks

**Review by zhangxuhong — APPROVED** [review]
